test/orchard/info_test.clj

```diff
@@ -99,24 +100,30 @@
     (let [target-ns 'orchard.info
           ns-syms (keys (ns-map target-ns))
           info-specials (fn [info-params]
-                          (as-> (map info/info* info-params) it
-                            (group-by :special-form it)
-                            (get it true)
-                            (map :name it)
-                            (set it)))]
+                          (let [{specials true} (->> info-params
+                                                     (map info/info*)
+                                                     (group-by :special-form))]
+                            (into #{}
+                                  (map :name)
+                                  specials)))]
       (when cljs-available?
         (testing "- :cljs"
           (let [special-doc-map (misc/require-and-resolve 'cljs.repl/special-doc-map)
-                special-syms (into '#{in-ns load load-file} (keys special-doc-map))]
```

Comment:
I removed `load-file` because it's a public function (https://github.com/clojure/clojurescript/blob/17068fa91cae169ed7c881adb593d85138efe6d0/src/main/cljs/cljs/core.cljs#L338) _and_ a special form (https://cljs.github.io/api/cljs.core/load-file)

Because that's such an exceptional case, for a rarely-used piece of functionality, I didn't spend much time deliberating whether to favor one or the other.